### PR TITLE
feat(metrics): longhorn_disk_status

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/6858
https://github.com/longhorn/longhorn/issues/6873
https://github.com/longhorn/longhorn/issues/6872

### CODE REVIEW

**Comment by c3y1huang** [comment]
cc @jillian-maroket , @innobead

**Review by innobead — APPROVED** [review]

File: content/docs/1.4.4/monitoring/metrics.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ weight: 3
 | longhorn_disk_capacity_bytes | The storage capacity of this disk | longhorn_disk_capacity_bytes{disk="default-disk-8b28ee3134628183",node="worker-3"} 8.3987283968e+10 |
 | longhorn_disk_usage_bytes | The used storage of this disk | longhorn_disk_usage_bytes{disk="default-disk-8b28ee3134628183",node="worker-3"} 9.060941824e+09 |
 | longhorn_disk_reservation_bytes | The reserved storage for other applications and system on this disk | longhorn_disk_reservation_bytes{disk="default-disk-8b28ee3134628183",node="worker-3"} 2.519618519e+10 |
+| longhorn_disk_status | The status of this disk | longhorn_disk_status{condition="ready", disk="default-disk-ca0300000000"} |
```
Comment (innobead) [inline content/docs/1.4.4/monitoring/metrics.md:41]: Why there is no node label in the example?

**Review by innobead — CHANGES_REQUESTED** [review]

**Comment by innobead** [comment]
cc @jillian-maroket when reviewing PRs, if the PRs related to the upcoming release, we should treat them as high priority.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by chriscchien — APPROVED** [review]
LGTM

**Review by innobead — APPROVED** [review]

**Pull request merged**